tutorials/asr/ASR_with_NeMo.ipynb

```diff
@@ -0,0 +1,920 @@
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "inRJsnrz1psq",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Start training!!!\n",
+        "trainer.fit(first_asr_model)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jpYXX-GslR0E",
+        "colab_type": "text"
+      },
+      "source": [
+        "There we go! We've put together a full training pipeline for the model and trained it for 50 epochs. Let's start Tensorboard to see how training went. Note that WER stands for Word Error Rate and the lower it is, the better.\n",
+        "You can definetely imporove this run by playng with hyperparametrers."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7kdQbpohXnEd",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Speaking of hyper parameters. Here they are:\n",
+        "print(params['model']['optim'])"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "AbigFKUtYgvn",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# To change, for example, learning rate, do:\n",
+        "import copy\n",
+        "new_opt = copy.deepcopy(params['model']['optim'])\n",
+        "new_opt['lr'] = 0.001\n",
+        "quartznet.setup_optimization(optim_config=DictConfig(new_opt))\n",
+        "# And then you can invoke trainer.fit(first_asr_model)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "n_0y3stSXDX_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Let's start Tensorboard to see how our training went.\n",
+        "%load_ext tensorboard\n",
+        "%tensorboard --logdir lightning_logs/"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PSJeNwkTXQjA",
+        "colab_type": "text"
+      },
+      "source": [
+        "You can use your freshly trained model to transcribe audio in much the same way as you did with cloud model. (Though full-fledged model from cloud is, of course, better)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3FT0klSV268p",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "print(first_asr_model.transcribe(paths2audio_files=['./an4/wav/an4_clstk/mgah/cen2-mgah-b.wav']))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2f142kIQc1Z2",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Transfer learning\n",
```

Comment:
I think this could go under the "Model Improvements" section.

Reply:
agree - moved